### ISSUE TITLE
Performance Test page Scan Options not working as intended

### ISSUE THREAD

**Issue by kojuharov** [Rarst/sg-cachepress#31]
https://sgmstest-subdomain.com/notmu/wp-admin/admin.php?page=performance-test

Currently if you go to the Performance Test page, and select whichever of the two options - Default or Advanced scan, the result is the same on the first try.

When you select Default scan, you should see what you see now.

However when you select Advanced Scan you should see the options allowing you to manage the scanned URLs and choose whether you want to test a logged in or not user.

The option to choose these things only shows if you run an Advanced scan once, but they should show when you only select the Advanced Scan radio button without the need to run a scan first.

Basically - make the Advanced Scan options show up when you select Advanced scan without needing to run the Advanced scan first.

**Comment by Rarst** [comment]
Oops :sweat: I've changed something for the network admin page and that caused script to be dequeued on performance test page. Same thing for all three issues, should all work now that script is back!
